lib/Dialect/TorchConversion/Transforms/Passes.cpp

```diff
@@ -101,6 +102,9 @@ void TorchConversion::createTorchBackendToTosaBackendPipeline(
     pm.addNestedPass<FuncOp>(createCSEPass());
   }
 
+  // Add the ToStandard pass for lowering some ops
+  pm.addNestedPass<FuncOp>(createTosaToStandard());
```

Comment:
Linking is failing for me with this change. Shouldn't `MLIRTosaToStandard` be added to `LINK_LIBS` of `lib/Dialect/TorchConversion/Transforms/CMakeLists.txt`?

Reply:
Can you tell me which target you are trying to build? 'check-torch-mlir' did not need this and all the checks here passed as well. @vivekkhandelwal1 created Issue #521 to track this.

Reply:
I see it for `ninja bin/torch-mlir-lsp-server` - the same as #521. The `lib/Dialect/TorchConversion/Transforms/CMakeLists.txt` file should really list all the libraries it directly depends on so that linking is deterministic and not dependent on the chance that `cmake` will build the lining order in a way that happens to link fully. If you add `MLIRTosaToStandard` to `LINK_LIBS` you will see that `cmake` adds `libMLIRTosaToStandard.a` after `libTorchMLIRTorchConversionPasses.a`. As in this link `libTorchMLIRTorchConversionPasses.a` is the first to require `createTosaToStandard()`, this order is required. I know `libMLIRTosaToStandard.a`  appears before `libTorchMLIRTorchConversionPasses.a` too, but the linker won't look into preceding `.a` files for newly unresolved symbols.

Reply:
For now, I have reverted the change to Passes.cpp. Will add it to <Transforms>/CMakeLists.txt when we renable this.